src/m365/aad/commands/app/app-add.ts

```diff
@@ -726,19 +762,11 @@ class AadAppAddCommand extends GraphCommand {
               }
             });
           }
-          
-          if (typeof this.manifest?.requiredResourceAccess !== 'undefined' && this.manifest.requiredResourceAccess.length > 0) {
+          else if (this.manifest.requiredResourceAccess.length > 0) {
```

Comment:
Since we've checked the length already when entering this method, would simple `else` suffice here?